### PR TITLE
Update DataFramesMeta blurb

### PR DESCRIPTION
**Pull request by pdeffebach**

### CODE REVIEW

File: docs/src/man/querying_frameworks.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ The major benefit of the package is that it allows you to refer to columns of a
 expressions. Additionally you can chain a sequence of transformations of a
 `DataFrame` using the `@linq` macro.
 
+When a DataFramesMeta macro such as `@select`, `@transform`, `@by`, `@combine`,
```
Comment (bkamins) [inline docs/src/man/querying_frameworks.md:27]: In DataFrames.jl we add .jl everywhere 😄 
```suggestion
When a DataFramesMeta.jl macro such as `@select`, `@transform`, `@by`, `@combine`, 
```

**Comment by bkamins** [comment]
Nice that all else works. Can you also please update line https://github.com/JuliaData/DataFrames.jl/pull/2561/files#diff-bf486009121d3fe05fafd9375068dc724f89e44524d86f520e773c9933dd0b75R12 to refer to the current version of the package?

**Review by bkamins — CHANGES_REQUESTED** [review]
Also https://github.com/JuliaData/DataFrames.jl/pull/2561/files#diff-bf486009121d3fe05fafd9375068dc724f89e44524d86f520e773c9933dd0b75R94 produces a different row order (`transform` retains the original row order)

**Comment by pdeffebach** [comment]
Sorry about that. should be fixed now.

**Comment by bkamins** [comment]
> Sorry about that.

No problem. I know the pain of making PRs (and then @nalimilan makes a review and I often wonder how could I make some errors that he spots 😄).

Thank you!

**Review by bkamins — APPROVED** [review]

**Pull request merged**